Update ja.json & Swipe gestures indicators

LEoREo2247 opened the pull request:
I just filled missing translation and fixed some very small translation error and added horizontal swipe gestures indicators like chrome

PalmerAL commented on css/webviews.css:
```diff
@@ -32,3 +32,40 @@ body.windows #webviews {
+	width: 140px;
+	height: 140px;
+	border-radius: 70px;
+	background: rgba(0,0,0,0.8);
```
Since there's another rule below to set the opacity, can this just be ```#000```?

PalmerAL commented on css/webviews.css:
```diff
@@ -32,3 +32,40 @@ body.windows #webviews {
+	position: fixed;
+	top: 50%;
+	z-index: 1;
+	transition: opacity 0.3s, visibility 0.3s;
```
There's no transition for ```visibility```, so I think that part can be removed without affecting anything.

PalmerAL commented on css/webviews.css:
```diff
@@ -32,3 +32,40 @@ body.windows #webviews {
+}
+
+.arrow {
+	color: white;
```
The indentation here doesn't match the other lines

PalmerAL commented on index.html:
```diff
@@ -61,7 +61,14 @@
 
-	<div id="webviews"></div>
+	<div id="webviews">
+		<div id="left" class="arrow-indicator">
```
Can you make the id's of these something more descriptive, like "leftArrowContainer"?

PalmerAL commented on css/webviews.css:
```diff
@@ -32,3 +32,40 @@ body.windows #webviews {
+	visibility: hidden;
+}
+.arrow-indicator.shown {  
+	opacity: 1;
```
I think it might be better to have this less opaque - maybe around 0.6?

PalmerAL commented on js/webviews.js:
```diff
@@ -274,6 +274,22 @@ webviews.bindIPC('goForward', function () {
+    backArrow.classList.toggle('shown')
+    setTimeout(function() {
+      backArrow.classList.toggle('shown')
+    }, 800)
```
800 seems a little too long, maybe somewhere around 500-600?

PalmerAL commented:
I've added a few minor comments. Otherwise, this looks pretty good!

LEoREo2247 commented:
Alright, I've updated my code based on feedback.

```css
.arrow-indicator {
	width: 140px;
	height: 140px;
	border-radius: 70px;
	background: #000;
	position: fixed;
	top: 50%;
	z-index: 1;
	transition: opacity 0.3s, visibility 0.3s;
	opacity: 0;
	visibility: hidden;
}
```
Though, `visibility` seems to be needed. Otherwise, arrow indicator's fade-out animation won't be applied.
